src/components/pages/parameter-or-variable.jsx

```diff
@@ -34,7 +38,7 @@ const ParameterOrVariablePage = React.createClass({
     }
     return (
       <div>
-        <Link className="btn btn-default" to="/">Retour à l'accueil</Link>
+        <Link className="btn btn-default" onClick={this.handleBackButtonClick} to="/">Retour</Link>
```

Comment:
Well, the current behaviour is worse IMO: on hover, my status bar indicates a misleading destination! You are dynamically hijacking the stated destination.

![capture d ecran 2017-02-27 a 10 38 38](https://cloud.githubusercontent.com/assets/222463/23356119/0036aace-fcd9-11e6-8acb-ebb9ac355b4e.png)

The best would be to interpolate the URL directly into the `href` attribute.